dist/freeciv-server.appdata.xml.in

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="console-application">
-    <id type="desktop">net.longturn.freeciv_server</id>
+    <id type="desktop">net.longturn.freeciv21.server</id>
```

Comment:
freeciv21_server

Reply:
I guess we don't need to differentiate the server in any other way. `freeciv21.server_console` seems superfluous.